src/renderer/components/AircraftSection/styles.tsx

```diff
@@ -8,9 +8,9 @@ export const Container = styled.div`
 `;
 
 export const HeaderImage = styled.div`
-      height: 400px;
-  background-image: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.6)), url('https://nyc3.digitaloceanspaces.com/fselite/2020/11/123263426_126778999193686_7966913238295950901_o.png');
-  background-size: cover;
+    height: 400px;
+    background-image: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.6)), url('https://cdn.discordapp.com/attachments/770835189419999262/789223040990052472/Screenshot_213.png');
```

Comment:
The resolution of this image is too big, it takes a couple of seconds to load. We can keep it and will switch to have the images locally.